frontend/amundsen_application/static/js/pages/TableDetailPage/WriterDropdown/styles.scss

```diff
@@ -0,0 +1,57 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+
+$writer-dropdown-padding-top: 5px;
+$writer-dropdown-max-size: 300px;
+$menu-title-font-size: 12px;
+$menu-item-row-gap: 15px;
+$menu-item-content-height: 18px;
+
+.writer-dropdown {
+  margin-right: $spacer-half !important;
+
+  .writer-dropdown-button {
+    border: 0;
+    padding: $writer-dropdown-padding-top $spacer-1;
+  }
+
+  .writer-dropdown-menu {
+    max-width: $writer-dropdown-max-size;
+    max-height: $writer-dropdown-max-size;
+    overflow-y: scroll;
+    line-height: normal;
+  }
+
+  .writer-dropdown-menu-title {
+    margin: $spacer-2 $spacer-2 0 $spacer-2;
+    line-height: $spacer-2;
+    text-transform: uppercase;
+    font-size: $menu-title-font-size;
+    white-space: nowrap;
+  }
+
+  .writer-dropdown-menu-item {
+    &.last-item {
```

Comment:
You might not need to add this class by using https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child, give it a try!